lib/Support/raw_ostream.cpp

```diff
@@ -456,6 +485,19 @@ raw_ostream &raw_ostream::operator<<(const FormattedNumber &FN) {
   }
 }
 
+// HLSL Change Starts - Add handling of numerical base IO manipulators.
+raw_ostream &raw_ostream::
+operator<<(std::ios_base &(*iomanip)(std::ios_base &)) {
+  if (iomanip == std::hex)
```

Comment:
We are using `std::hex` & `std::oct` here. Need to `#include <ios>` at the beginning. Compiling towards macOS will error out if the header is missing.